Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
```

Comment:
I would not be whole against this, but since Rectangles don't suppose Interpolation at this time, it seems generous to elevate them to prestige of having an Enum named after them. If it is anticipated that they may somehow support interpolation in the future (perhaps with vast numbers of rectangles... this doesn't sound healthy though) then sorting the interface out now would be sensible.